src/modules/feature-modules/accessor/pages/innovation/support/organisations-support-status-suggest.component.html

```diff
@@ -27,21 +42,47 @@
       <div class="nhsuk-grid-row">
         <div class="nhsuk-grid-column-two-thirds">
-          <p class="nhsuk-u-font-weight-bold nhsuk-u-margin-0">You have selected:</p>
-          <ul class="nhsuk-list nhsuk-list--bullet nhsuk-u-margin-bottom-5">
-            <li *ngFor="let organisation of chosenUnits.list">
-              {{ organisation.organisation }}
-              <ul *ngIf="organisation.units.length > 0" class="nhsuk-list nhsuk-list--bullet nhsuk-u-margin-0">
-                <li *ngFor="let unit of organisation.units" class="nhsuk-u-margin-0">{{ unit }}</li>
+          <div class="bg-color-white nhsuk-u-padding-top-3 nhsuk-u-padding-bottom-3 nhsuk-u-margin-bottom-5">
+            <ng-container *ngIf="chosenUnits.unitsNames.length > 0; else noUnits">
+              <p class="nhsuk-u-font-weight-bold">You have suggested {{ chosenUnits.unitsNames.length > 1 ? "these units" : "this unit" }}:</p>
+              <ul
+                class="nhsuk-list nhsuk-list--bullet nhsuk-u-margin-left-3"
+                [ngClass]="chosenUnits.organisation.information ? 'nhsuk-u-margin-bottom-4' : 'nhsuk-u-margin-bottom-0'"
+              >
+                <li *ngFor="let name of chosenUnits.unitsNames">
+                  {{ name }}
+                </li>
               </ul>
-            </li>
-          </ul>
+            </ng-container>
+            <ng-template #noUnits>
+              <p class="nhsuk-u-font-weight-bold" [ngClass]="{ 'nhsuk-u-margin-bottom-0': !chosenUnits.organisation.information }">
+                You have selected {{ chosenUnits.organisation.name }} ({{ chosenUnits.organisation.acronym }})
+              </p>
+            </ng-template>
 
-          <theme-form-textarea controlName="comment" label="Describe briefly why you selected these organisations" [pageUniqueField]="false" lengthLimit="xl"></theme-form-textarea>
-          <theme-form-checkbox controlName="confirm" label="I confirm that I understand the support offers of the selected organisations" size="small"></theme-form-checkbox>
+            <ng-container *ngIf="chosenUnits.organisation.information" class="nhsuk-margin-bottom">
+              <ng-container *ngFor="let supportInfo of chosenUnits.organisation.information.supportInformation; last as lastItem">
+                <p class="font-color-secondary nhsuk-u-margin-bottom-1">{{ supportInfo.title }}</p>
+                <ul class="nhsuk-list nhsuk-list--bullet font-color-secondary nhsuk-u-margin-left-3" [ngClass]="lastItem ? 'nhsuk-u-margin-bottom-0' : 'nhsuk-u-margin-bottom-4'">
+                  <li *ngFor="let supportInfoBulletPoint of supportInfo.bulletPoints">
+                    {{ supportInfoBulletPoint }}
+                  </li>
+                </ul>
+                <p *ngIf="supportInfo.footnote" class="font-color-secondary">{{ supportInfo.footnote }}</p>
+              </ng-container>
+            </ng-container>
+          </div>
+
+          <theme-form-textarea
+            [id]="'comment'"
```

Comment:
```suggestion
            id="comment"
```

Reply:
I did it like you said, but then I saw it add the same id to the text area component as well (print)
![image](https://github.com/nhsengland/innovation-service-transactional-frontend/assets/124596230/972dbbf5-b61e-49e6-9d20-59f0aa2d2d25)

Reply:
Oh, makes sense now! Is because it has an input `id`! Thanks!